server/src/main/webapp/WEB-INF/rails/webpack/models/roles/roles_crud.ts

```diff
@@ -19,7 +19,7 @@ import {SparkRoutes} from "helpers/spark_routes";
 import {BulkUserRoleUpdateJSON, GoCDRole, PluginRole, Role, Roles} from "models/roles/roles";
 
 export class RolesCRUD {
-  private static API_VERSION_HEADER = ApiVersion.v2;
+  private static API_VERSION_HEADER = ApiVersion.v3;
```

Comment:
use latest?

Reply:
My initial thought as well but in all the SPAs we use targeted versions. Should I use latest?

Reply:
Hmm, with SPAs, it's tricky.
As if we use `latest`, A change in the API can possibly cause the SPA to break (unintentionally).

Let's use the targeted version instead of latest!